exercises/chapter 10/chapter10.Rmd

```diff
@@ -15,6 +15,16 @@ Before diving into the exercises, follow the instructions below to download a 20
 -   Upload the file to your RStudio. 
 
 
+## Load Necessary Libraries
+
+Before starting the exercises, ensure you have the necessary libraries loaded:
+
+```{r}
+# Load necessary libraries
+library(tidyverse)
+
```

Comment:
Delete the unnecessary blank line here, and in all similar cells across the exercises